### ISSUE TITLE
releaseRevision collides with bug in helm-diff

### ISSUE THREAD

**Issue by sjentzsch** [giantswarm/starboard-exporter#43]
Question: Could you remove the `releaseRevision` annotation in the starboard-exporter Deployment? (https://github.com/giantswarm/starboard-exporter/blob/main/helm/starboard-exporter/templates/deployment.yaml#L19)

There is a bug in helm-diff (https://github.com/databus23/helm-diff/issues/253) not being able to properly cope with this.
We are currently using a diff-based deployment on our end, and this chart is thus always shown as a helm chart with changes to be deployed.
Actually, we have not seen this issue just until now, and we are deploying a whole bunch of helm charts; haven't seen this being added to the annotation actively anywhere else. Is there any point in adding the `releaseRevision` as an annotation?

**Comment by stone-z** [comment]
Hey @sjentzsch, I'm removing the annotation in https://github.com/giantswarm/starboard-exporter/pull/44.

For background, we used to set the `releaseRevision` for change detection purposes internally (e.g. to selectively roll pods or not when some configuration changes out-of-band). We've since stopped using that method but this was generated from a template which still includes it for legacy reasons.
